docs/mkdocs.yml

```diff
@@ -87,6 +87,9 @@ nav:
           - ./examples/agent/llm_compiler.ipynb
           - ./examples/agent/structured_planner.ipynb
           - ./examples/agent/introspective_agent_toxicity_reduction.ipynb
+          - Memory:
+              - ./examples/agent/memory/vector_memory.ipynb
+              - ./examples/agent/memory/composable_memory.ipynb
```

Comment:
@logan-markewich: also, would this successfully create a subsection under "Agents" (within examples)? Do we want to do something like this, or should we place these notebooks elsewhere?

Reply:
This should work yes 👍 although this section is built automatically if you run `python docs/prepare_for_build.py` (it picks up all the example notebooks by reading the file structure)

Reply:
aha. good to know! thanks :)